src/commands/Member/IndividualRanking.ts

```diff
@@ -0,0 +1,97 @@
+        const memberRes = (await Members.findAll({ where: {discord_client_id: clientId} }))[0];
+        if (!memberRes) {
+            interaction.reply({ content: "Something went wrong. Please `/logout` and re-verify your account again, or contact an admin for help.", flags: MessageFlags.Ephemeral });
+            return;
+        }
+        const membershipId = memberRes.dataValues.membership_id;
+        if (!membershipId) {
+            interaction.reply({ content: "You have not registered for the current semester. Use `/register` to register.", flags: MessageFlags.Ephemeral });
+            return;
+        }
+
+        const recordedMembershipSemesterId : string = (await uwpscApiAxios.get(`/memberships/${membershipId}`)).data.semesterId;
+        if (recordedMembershipSemesterId != currentSemesterId) {
+            interaction.reply({ content: "You have not registered for the current semester. Use `/register` to register.", flags: MessageFlags.Ephemeral });
+            return;
+        }
+
+
+        const rankingRes = (await uwpscApiAxios.get(`/semesters/${currentSemesterId}/rankings/${membershipId}`));
+        if (rankingRes.status != 200) {
+            // This would only run if the player registered for the current semester, but has yet to play an event
+            interaction.reply({ content: "No records of your performance this term can be found.", flags: MessageFlags.Ephemeral });
+            return;
+        }
+
+        const currentSemesterName = currentSemesterConfigRes.dataValues.current_semester_name;
+        const userRegisteredName: string = `${memberRes.dataValues.first_name} ${memberRes.dataValues.last_name}`;
+        const userId: number = memberRes.dataValues.user_id;
+        const position: string = rankingRes.data.position;
+        const points: string = rankingRes.data.points;
+        let color: ColorResolvable;
+        if (parseInt(position) <= 100) {
+            color = [10, 149, 72];
+        } else if (parseInt(position) <= 120) {
+            color = [229,162,103];
+        } else {
+            color = [163, 0, 0];
+        }
+
+        const avatarUrl: string = interaction.user.displayAvatarURL()
+        const memberProfileEmbed = new EmbedBuilder()
+            .setTitle(`${currentSemesterName} Performance`)
+            .setAuthor({ name: `${interaction.user.displayName}`, iconURL: `${avatarUrl}`})
+            .setColor(color)
+            .setThumbnail(this.client.config.misc.logoUrl)
+            .addFields(
+                { name: "Name", value: `${userRegisteredName}`, inline: true },
+                { name: "ID", value: `${userId}`, inline: true },
+                { name: "\u200b", value: "\u200b" },
+                { name: "Position", value: `${position}`, inline: true },
+                { name: "Points", value: `${points}`, inline: true },
+            );
+
+        interaction.reply({ embeds: [memberProfileEmbed], flags: MessageFlags.Ephemeral });
```

Comment:
Can we only display the position and points. If the message is only seen by the user using the command then they don't need to see their name. This also probably means we don't need their name stored in the database. So you can remove that code as well.